hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/TableResolverImpl.java

```diff
@@ -272,14 +273,22 @@ public List<Table> getTables() {
 
     private Table toTable(Mapping mapping) {
         SqlConnector connector = connectorCache.forType(mapping.type());
-        return connector.createTable(
-                nodeEngine,
-                SCHEMA_NAME_PUBLIC,
-                mapping.name(),
-                mapping.externalName(),
-                mapping.options(),
-                mapping.fields()
-        );
+        try {
+            return connector.createTable(
+                    nodeEngine,
+                    SCHEMA_NAME_PUBLIC,
+                    mapping.name(),
+                    mapping.externalName(),
+                    mapping.options(),
+                    mapping.fields()
+            );
+        } catch (QueryException e) {
```

Comment:
Perhaps we should catch `Throwable`, it's calling user code (not yet, but it should). There can be poor quality error-producing connector, it shouldn't break the whole engine. The mapping can be broken in any way, not just missing class.